drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -183,7 +183,7 @@ static int kinetis_usb_init(void)
 
 	irq_enable(CONFIG_USBD_KINETIS_IRQ);
 
-	SYS_LOG_DBG("");
+	USB_DBG("");
```

Comment:
what's the purpose of such empty message?

Reply:
The meaning of this is to print something in a defined place (like tracing with SYS_LOG), usually it is just the name of the function.